.gitleaks.toml

```diff
@@ -0,0 +1,5 @@
+[[allowlists]]
+    # Exclude a specific file
+    paths = [
+        '''(?:^|/)htdocs/includes/php-jwt/README\ .md$''',
```

Comment:
The `\` is for the dot, the space should be removed.

Reply:
aïe